airflow/providers/pinecone/operators/pinecone.py

```diff
@@ -119,7 +119,7 @@ def __init__(
         pod_type: str | None = None,
         metadata_config: dict | None = None,
         source_collection: str | None = None,
-        metric: str | None = None,
+        metric: str | None = "cosine",
```

Comment:
```suggestion
        metric: str = "cosine",
```

Reply:
changed it

Reply:
what did we change exactly?

Reply:
I checked it from the upstream library. It defaults to `cosine`. Hence keeping it back to `cosine`.

Reply:
Also metric value can be `None` as well

Reply:
Same question here. If `None` is an acceptable value, then why are we defaulting to "cosine"?

Reply:
The upstream library aspects default to `cosine` as we had discussed on the call. Hence just setting it to default as discussed

Reply:
Not very clear yet. But happy with the default to cosine